### ISSUE TITLE
Please explain plugin usefulness

### ISSUE THREAD

**Issue by pcav** [giscan/Generalizer#1]
The functionality described in the About seems already present in core QGIS Processing.
Please clarify what the plugin is actually adding.
Sorry for bothering, but we are trying hard not to duplicate functionality, so to reduce confusion for the users and administrative burden.
Thanks.

**Comment by kikislater** [comment]
Hi @pcav 

Goals of this plugin is :

* Major of functions were rewritten by Piotr in python and does not need grass (even if it's originally written against grass addon).

* Settings are predifined to obtain good results out of the box so it's a simplier interface to 

* Plugin exists since Qgis 1.6 or 1.8. It was asked by previous Qgis users when they moved to Qgis 3

Regards,

**Comment by pcav** [comment]
Is this any different from the existing alg in Processing/core?

**Comment by kikislater** [comment]
It's full python that's the main difference (doesn't require grass). Otherwise it's the same !

**Comment by pcav** [comment]
I'm talking about the core alg, not the grass one. If it's really the same, I'd suggest dropping this one. I know it's a shame wasting work done, but duplication has to have a reason. I hope you'll understand.

**Comment by kikislater** [comment]
OK so you're talking about smoothing and simplify which are now in processing ? 
Didn't know there was these processing 
Plugin do a little more but ok we can close. I will push on my own repo

**Comment by pcav** [comment]
If the plugin has more options it makes sense to keep it. Of course the best option would be to add these options to core function.
Please do not hesitate asking for help if necessary.
Thanks.

**Comment by pcav** [comment]
Sorry, I forgot about this. I'll publish it because of the added functions.
Thanks.

**Comment by kikislater** [comment]
@pcav Well, may be I made a mistake. Plugin already exists but I couldn't edit as original founder is maintainer but don't answer to message ... That's why I made it compatible to qgis 3 ! 
Could you arrange or change maintainer ?

**Comment by pcav** [comment]
currently:
 Maintainer
    sylvain

**Comment by kikislater** [comment]
Here is the mistake : 

https://plugins.qgis.org/plugins/generalizer/

and 

https://plugins.qgis.org/plugins/Generalizer3/

**Comment by pcav** [comment]
sorry, it's unclear to me. you are the maintainer of 
https://plugins.qgis.org/plugins/Generalizer3/
what should be changed?
merci

**Comment by kikislater** [comment]
Sorry, I will explain further : 

This plugin was originally created years ago and is available until qgis 2 here : https://plugins.qgis.org/plugins/generalizer/
All details remain in this plugin link : original repo, original contributor etc ...

As long as original contributor don't answer to message to upgrade to qgis 3, I made some modifications to make it compatible to qgis 3 but I couldn't publish as same name. So I publish with number 3 at the end of original plugin. 

Some of my clients were using this plugin and they were asking if there is a way to have it in qgis3. As it just only minor modifications, I did it freely and then tend to publish it ... Hope it's more clever now. If still in doubt, don't hesitate to revert back from me.
